File: LeetCode/Csharp/9-Palindrome-Number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        if(x<0) return false;
        
        var rev = 0;
        var original = x;
        var index = 1;
        while(original>0)
        {
            var remainder = original%10;
            rev = (rev*10) + remainder;
			
            original = original/10;
        }
        
		
        return rev == x;
    }
}

File: LeetCode/C++/9-Palindrome-Number.cpp
class Solution {
public:
bool isPalindrome(int x)
{
    long reverse = 0;
    long original = x;

    while(original > 0)
    {
        long remainder = original % 10;
        reverse = (reverse*10) + remainder;

        original = original / 10;
    }

    return (reverse == x);
}
};